tavla/app/(admin)/components/Preview/index.tsx

```diff
@@ -1,21 +1,112 @@
 'use client'
 
+import { LeftArrowIcon, RightArrowIcon } from '@entur/icons'
 import { Board } from 'Board/scenarios/Board'
-import { useEffect, useState } from 'react'
+import { usePostHog } from 'posthog-js/react'
+import { SetStateAction, useEffect, useState } from 'react'
 import { TBoard } from 'types/settings'
 
+const CarouselIndicators = ({
+    boards,
+    activeIndex,
+    onClick,
+}: {
+    boards: TBoard[]
+    activeIndex: number
+    onClick: (index: number) => void
+}) => {
+    return (
+        <div className="flex flex-row md:space-x-3 space-x-5 justify-center h-[10vh]">
+            {boards.map((_, index) => (
+                <button
+                    key={index}
+                    className={`md:w-5 md:h-5 w-6 h-6 rounded-full  bottom-5 mt-[2vh] left-1/2 ${
+                        index === activeIndex ? 'bg-blue' : 'bg-tertiary'
+                    }`}
+                    onClick={() => onClick(index)}
+                />
```

Comment:
Har vi sjekket hvordan dette høres ut for skjermlesere? Tror kanskje vi må legge på en title eller aria-label for å indikere hva knappene gjør

Reply:
sjekka nå, knappene funker greit men selve preview av boards er helt krise, går nesten ikke an å komme seg forbi den og så videre til teksten under fordi den roterer av seg selv

Reply:
har fiksa